tests/test_runner/test_debug_sql.py

```diff
@@ -74,14 +74,14 @@ def _test_output(self, verbosity):
         return stream.getvalue()
 
     def test_output_normal(self):
-        full_output = self._test_output(1)
+        full_output = self._test_output(1).replace("\n", " ")
         for output in self.expected_outputs:
             self.assertIn(output, full_output)
         for output in self.verbose_expected_outputs:
             self.assertNotIn(output, full_output)
 
     def test_output_verbose(self):
-        full_output = self._test_output(2)
+        full_output = self._test_output(2).replace("\n", " ")
```

Comment:
Thanks for the patch @Giebisch, I think we should adjust the `expected_outputs` instead to be properly formatted in order to ensure formatting is actually used.

https://github.com/django/django/blob/78657a201308a0b53661e2b267914407a9fd7fa1/tests/test_runner/test_debug_sql.py#L90-L111

https://github.com/django/django/blob/78657a201308a0b53661e2b267914407a9fd7fa1/tests/test_runner/test_debug_sql.py#L124-L133

The adjustments made make the tests pass with and without the proposed changes.